lib/tasks/dev.rake

```diff
@@ -6,7 +6,9 @@ if Rails.env.development? || Rails.env.test?
     task prime: "db:setup" do
       include FactoryGirl::Syntax::Methods
 
-      # create(:user, email: "user@example.com", password: "password")
+      create(:redirection, slug: "adarsh", url: "http://adarsh.io")
```

Comment:
They get linked in the factory: https://github.com/gabebw/hotline-webring/blob/master/spec%2Ffactories.rb#L9